### PR TITLE
Adding a void function that prints errors 

### PR DESCRIPTION
**Pull request by abt2804**
This is a void function that has a char string that prints specific errors if the user do something wrong. It also makes easier to add new error messages while adding new features.

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -72,6 +72,12 @@ int CalcInit(void)
     return 0;
 }
 
+/*A void function that wil print errors for the user*/
```
Comment (GeoB99) [inline main.c]: Can you put spaces between the comment syntax slashes?

File: main.c [hunk 2]
```diff
@@ -72,6 +72,12 @@ int CalcInit(void)
...
+void printError(char stringError[])
+{
+	PrintString("Error! : %s! The application will now exit...\n", stringError);
```
Comment (GeoB99) [inline main.c]: I'd rather not have the exclamation point hard-coded in the string format. It's really not intuitive into doing so. Furthermore, could you please capitalize the first letter of the routine's name (just like every function here in the code)?

**Comment by abt2804** [comment]
@Fraizeraust I was kinda busy for 3 days but hey I am back! :)

**Review by GeoB99 — APPROVED** [review]
LGTM.

The only thing to do now is to remove the redundant colon and it'll be ready for merge.

**Comment by abt2804** [comment]
Here you go!

**Pull request merged**